doc/rados/operations/erasure-code.rst

```diff
@@ -108,7 +117,11 @@ no two *chunks* are stored in the same rack.
               +----------------->| OSD5 |
                                  +------+
 
- 
+Because six racks are available, but objects are each spread across
```

Comment:
>  Because six racks are available, but objects are each spread across five

Remove the comma:

"Because six racks are available but objects are each spread across five,"

Why should the comma be removed?

Remove the comma here so that the sentence is balanced between the condition of the cluster in the first part and the implications of that condition in the second part.